src/test/resources/expected/test-hooks-10.kore.golden

```diff
@@ -0,0 +1,97 @@
+Definition
+    { definitionAttributes = Attributes []
+    , definitionModules =
+        [ Module
+            { moduleName = ModuleName "TEST-HOOKS-10"
+            , moduleSentences =
+                [ ObjectSentence (SentenceHookSentence (SentenceHookedSort SentenceSort
+                    { sentenceSortName = (Id "S" AstLocationNone) :: Id Object
+                    , sentenceSortParameters = []
+                    , sentenceSortAttributes =
+                        Attributes 
+                            [ Fix (UnifiedPattern (UnifiedObject (Rotate31 (ApplicationPattern Application
+                                { applicationSymbolOrAlias =
+                                    SymbolOrAlias
+                                        { symbolOrAliasConstructor = (Id "hook" AstLocationNone) :: Id Object
```

Comment:
These lines are way too long.

Reply:
This is a golden file; what can we do about it?